### PR TITLE
Implement add-on installation/uninstallation.

### LINKED ISSUE #196: Install add-on
As a user I want to install additional adapters to those which shipped with the gateway

This may end up being as simple as `npm install` from the command line

### LINKED ISSUE #198: Uninstall add-on
As a user I want to uninstall an adapter I am not using

### PR DESCRIPTION
**Pull request by mrstegeman**
Fixes #196
Fixes #198

### CODE REVIEW

**Comment by mrstegeman** [comment]
@hobinjk @benfrancis could you take another look at this?

File: static/js/api.js [hunk 1]
```diff
@@ -133,93 +133,121 @@
...
-          reject(error);
-        });
+      var headers = {
+        'Authorization': `Bearer ${window.API.jwt}`,
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      };
+      var payload = {
+        'enabled': enabled
+      };
```
[the remark this answers is not in the document]
Comment (mrstegeman) [reply static/js/api.js]: It's using the function argument. :-)

File: static/images/remove.svg [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
[the remark this answers is not in the document]
Comment (mrstegeman) [reply static/images/remove.svg]: Done.

File: static/js/api.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (benfrancis) [reply static/js/api.js]: LOL, sorry.

**Pull request merged**